x-pack/solutions/security/plugins/security_solution/public/common/components/security_route_page_wrapper/index.tsx

```diff
@@ -22,6 +22,8 @@ interface SecurityRoutePageWrapperProps {
    * Used primarily in the AI for SOC tier, to allow redirecting to the home page instead of showing the NoPrivileges page.
    */
   redirectIfUnauthorized?: boolean;
+  // Used to disable the SpyRoute for the page if page's children have their own specified.
```

Comment:
I would be clearer here and use an explicit noun instead of an implied one:

```suggestion
  // Used to disable the SpyRoute for the page, if e.g. the page's children have their own SpyRoute specified.
```